File: College/2ª Semestre/Sistemas Embarcados/Exercícios/Linguagem C - Parte 3/Exercício01.c
#include <stdio.h>

int parOuImpar(int num)
{
    if (num % 2 == 0)
    {
        return 1;
    }
    else
    {
        return 0;
    }
}

int main()
{
    int n;
    printf("Digite um número: ");
    scanf("%d", &n);

    if (parOuImpar(n))
    {
        printf("Número par.\n");
    }
    else
    {
        printf("Número impar.\n");
    }
    
}

File: College/2ª Semestre/Programação de Computadores/3ª Lista de Exercícios (2S)/Exercício01.py
def parOuImpar(num):
    if num % 2 == 0:
        return 1
    else:
        return 0

num = int(input("Digite um número: "))

if parOuImpar(num):
    print("Número Par.")
else:
    print("Número Ímpar.")
